debian/control

```diff
@@ -1,10 +1,10 @@
 Source: kodi-visualization-fishbmc
 Priority: extra
-Maintainer: Arne Morten Kvarving <spiff@xbmc.org>
-Build-Depends: debhelper (>= 8.0.0), cmake, kodi-addon-dev,
+Maintainer: Nobody <nobody@kodi.tv>
```

Comment:
@notspiff Are this change of name here and on other places OK for you?

Do not just want to remove your good name without OK from you.